app/controllers/subscriber/group_controller.rb

```diff
@@ -82,5 +82,17 @@ def replace_validation_errors
       end
     end
 
+    def subscription_tags
+      [collect_tags, collect_tags(excluded: true)].flatten.map do |tag|
+        SubscriptionTag.new(subscription: entry,
+                            tag_id: tag[:id],
+                            excluded: tag[:excluded]) unless tag[:id].empty?
+      end.compact
+    end
+
+    def collect_tags(excluded: false)
+      param_key = excluded ? :excluded_subscription_tags : :included_subscription_tags
+      model_params[param_key]&.map { |tag| { id: tag, excluded: false } } || []
```

Comment:
create own method collect_excluded_tags ?

`{ id: tag, excluded: false }` shouldn't this be `{ id: tag, excluded: excluded }` ?